src/phases/replication/startup/StartupPhasesTemplate.yml

```diff
@@ -0,0 +1,237 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: |
+  Common definitions to support the workloads in replication/startup.
+
```

Comment:
Could you document add some documentation for all the parameters?

Reply:
Done

Reply:
Is there a push missing? I don't see any thing new here.

Reply:
See [latest commits](https://github.com/mongodb/genny/pull/952/commits/e23c88786726b77e820a507ff001a68db64a17ef)